server/ServerNetworking.cpp

```diff
@@ -188,7 +188,14 @@ PlayerConnection::~PlayerConnection() {
                                  << " for player id " << m_ID;
         }
     }
-    m_socket.close();
+    boost::optional<boost::asio::ip::tcp::socket> socket;
```

Comment:
Do you need this temporary if you move in the capture?

Reply:
Yep, I don't need it.